doc/tutorials/core/univ_intrin/univ_intrin.markdown

```diff
@@ -12,20 +12,177 @@ Universal Intrinsics {#tutorial_univ_intrin}
 Goal
 ----
 
-The goal of this tutorial is to provide a guide to using the Universal Intrinsics module to vectorize code for a faster runtime. 
-We'll briefly look into _SIMD intrinsics_ and how to work with wide _registers_, followed by a tutorial on how to use them to vectorize a convolution operation.
+The goal of this tutorial is to provide a guide to using the Universal Intrinsics module to vectorize your C++ code for a faster runtime.
```

Comment:
It would be better to use something like "functionality" or "feature" etc instead of "module" because UI is a part of OpenCV core module. So using UI doesn't add additional module dependencies to the code